### PR TITLE
fix(@schematics/angular): default newProjectRoot to current directory

### PR DESCRIPTION
**Pull request by clydin**
Fixes #13704

### CODE REVIEW

**Comment by FrozenPandaz** [comment]
@clydin FYI, this does not fix the breaking change which we discussed. There is still logic which expects `newProjectRoot` to be (or not to be) `undefined`.

**Comment by clydin** [comment]
`newProjectRoot` is only in a conditional which checks if it is truthy in the application schematic:
https://github.com/angular/angular-cli/blob/dad48c3af6bb2e3e94d7a29f0c70cf85400c1563/packages/schematics/angular/application/index.ts#L351

I updated the PR to maintain the existing behavior for that case.  Even though there is an argument to be made that it is actually a defect.  As is, the 'e2e' directory would overlap an existing 'e2e' folder at the workspace root and the schematic would fail in the case of the workspace's newProjectRoot being falsy.

**Comment by FrozenPandaz** [comment]
Awesome! Those changes fix the breaking change.

Thank you so much!

File: packages/schematics/angular/application/index.ts [hunk 1]
```diff
@@ -326,7 +326,7 @@ export default function (options: ApplicationOptions): Rule {
       };
 
     const workspace = getWorkspace(host);
-    let newProjectRoot = workspace.newProjectRoot || 'projects';
+    let newProjectRoot = workspace.newProjectRoot || '';
```
Comment (hansl) [inline packages/schematics/angular/application/index.ts]: Isn't that a breaking change? This seems to be 2 changes; a fix and a feature...

**Pull request merged**